Week1/homework/js-exercises/roundNumber.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+//declare a value
+const z = 7.25;
+
+//print the value of z
+console.log('value of Z is ' + z);
+
+//declare a value
+const a = 7;
+
+//printing the value
+console.log(`value of A is  ${Math.round(z)}`); // can be used Math.floor method
+
+// store the height value in new variable
+const storeHeightValue = Math.max(a, z);
```

Comment:
A better name for the variable could have been: `const maxHeightValue`. It's more expressive about the value it holds.

Reply:
i have implement that as well thanks for your advise , that help me to improve my coding